documentation/src/main/markdown/integrations/jenkinsplugin/autoescapingparameters.md

```diff
@@ -15,5 +15,7 @@ Jenkins enables you to set an environment variable at different levels, such as
 
 **Note:** In [solution_name] plugin version 9, the above recommendations remain the same for agents on Windows systems.  For those agents running on 'NIX systems, *DETECT\_PLUGIN\_ESCAPING* should be set to false.  Ensure that you adhere to the quoting conventions described above. Any input with spaces in the Jenkins configuration should be enclosed in quotes.
 
-[solution_name] for Jenkins allows some special characters when *DETECT\_PLUGIN\_ESCAPING* is set to false, and spaces can be included without escape sequences provided that they are enclosed in single or double quotes. Therefore, instead of `My\ Test\ Project1`, you can pass it as `'My Test Project1'`, the project will created and uploaded to [blackduck_product_name] as `My Test Project1*.*`
+[solution_name] for Jenkins allows some special characters when *DETECT\_PLUGIN\_ESCAPING* is set to false, and spaces can be included without escape sequences provided that they are enclosed in single or double quotes as described above for different agents. Therefore, instead of `My\ Test\ Project1`, you can pass it as `'My Test Project1'`, the project will created and uploaded to [blackduck_product_name] as `My Test Project1*.*`
+
+If *DETECT\_PLUGIN\_ESCAPING* is set to true, then you can use backtick (\`) to escape spaces which are passed within argument values. Therefore, you can pass values as "Windows` Project" in the arguments.
```

Comment:
Probably don't require the ", then" in the first sentence.
Can you explain a bit more of this line? _Therefore, you can pass values as "Windows` Project" in the arguments._
Is this meaning to provide an example, and would it be?: 
"As an example, you can pass a value like "Windows Project" by providing it on the command line as an argument in the format "Windows\` Project".

Reply:
Yeah it is weird how I can write this, so windows does not support escaping spaces with \ but ` instead. Therefore, I need to show an example how user can use backtick to escape spaces.